milatools/cli/utils.py

```diff
@@ -132,3 +134,15 @@ def qualified(node_name):
     if "." not in node_name and not node_name.endswith(".server.mila.quebec"):
         node_name = f"{node_name}.server.mila.quebec"
     return node_name
+
+
+def get_fully_qualified_name() -> str:
+    """Return the fully qualified name of the current machine.
+
+    Much faster than socket.getfqdn() on Mac. Falls back to that if the hostname command is not available.
+    """
+    try:
+        return subprocess.check_output(["hostname", "-f"]).decode("utf-8").strip()
+    except (FileNotFoundError, subprocess.CalledProcessError):
+        # Fall back, e.g. on Windows.
```

Comment:
```suggestion
    except Exception:
        # Fall back, e.g. on Windows.
```

Are these the only errors that can happen? I'm tempted to just catch Exception here because testing on Windows is a hassle and there isn't really any kind of exception we want to go through (except maybe KeyboardInterrupt which is a BaseException).